src/hooks/useKeyboardShortcut/keyCodes.ts

```diff
@@ -0,0 +1,210 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+export const modifierKeys = {
```

Comment:
See https://github.com/mediamonks/react-kit/pull/368/files#r1961612183

Reply:
Modifiers are not that sizable, and its values are used runtime in code (in `Object.values()`). Swapping them out with const enums would make the code less dynamic and potentially even larger.